std/bigint.d

```diff
@@ -1124,15 +1121,15 @@ public:
      * $(TR $(TD null) $(TD Default formatting (same as "d") ))
      * )
      */
-    void toString(scope void delegate(const (char)[]) sink, string formatString) const
+    void toString(Writer)(scope Writer sink, string formatString) const
```

Comment:
This is because a `@safe` function cannot call a `@system` sink and making the delegate `@safe` would be a breaking change.